airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftDestination.java

```diff
@@ -25,45 +25,59 @@
 package io.airbyte.integrations.destination.redshift;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import com.google.common.collect.ImmutableMap;
 import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.resources.MoreResources;
 import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.base.DestinationConsumer;
 import io.airbyte.integrations.base.IntegrationRunner;
-import io.airbyte.integrations.destination.jdbc.AbstractJdbcDestination;
-import java.util.Optional;
+import io.airbyte.integrations.destination.redshift.RedshiftCopyDestination.S3Config;
+import io.airbyte.protocol.models.AirbyteConnectionStatus;
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.ConnectorSpecification;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class RedshiftDestination extends AbstractJdbcDestination implements Destination {
+public class RedshiftDestination implements Destination {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(RedshiftDestination.class);
 
-  public static final String DRIVER_CLASS = "com.amazon.redshift.jdbc.Driver";
+  public static void main(String[] args) throws Exception {
+    final Destination destination = new RedshiftDestination();
+    LOGGER.info("starting destination: {}", RedshiftDestination.class);
+    new IntegrationRunner(destination).run(args);
+    LOGGER.info("completed destination: {}", RedshiftDestination.class);
+  }
 
-  public RedshiftDestination() {
-    super(DRIVER_CLASS, new RedshiftSQLNameTransformer(), new RedshiftSqlOperations());
+  @Override
+  public DestinationConsumer<AirbyteMessage> write(JsonNode config, ConfiguredAirbyteCatalog catalog) throws Exception {
+    if (hasCopyConfigs(config)) {
```

Comment:
might be good to DRY this if block into a method

Reply:
I played around with this and the other approach I came up with was DRY but more complicated and I like this simple form. I'm going to merge this in first to close out the ticket. Happy to iterate on this with you outside. Do you have a particular structure in mind?